_includes/video

```diff
@@ -9,5 +9,9 @@
   <iframe src="https://www.youtube-nocookie.com/embed/{{ video_id }}" frameborder="0" allowfullscreen></iframe>
 {% elsif video_provider == "google-drive" %}
   <iframe src="https://drive.google.com/file/d/{{ video_id }}/preview" frameborder="0" allowfullscreen></iframe>
+{% elsif video_provider == "bilibili" %}
+<div class="responsive-video-container">
```

Comment:
There's already a `<div class="responsive-video-container">` surrounding the whole `if` statement. Might be better to drop this duplicated `<div>` wrapper,

Reply:
Yes. You're right. I have deleted the  duplicated `<div>` wrapper.